### PR TITLE
Respond to PR #46 comments

### PR DESCRIPTION
**Pull request by Erin-Mounts**

### CODE REVIEW

File: BridgeAppSDK/DataArchiving/SBAActivityArchive.swift [hunk 1]
```diff
@@ -48,15 +60,13 @@ class SBAActivityArchive: SBADataArchive {
...
+                                assertionFailure("Unsupported archiveable result type: \(className)")
                             }
                         }
                     }
                 }
                 
-                if let error = archive.completeArchive() {
-                    print("Completing the archive of \(activityResult.schemaIdentifier) failed:\n\(error)")
-                } else {
+                if archive.completeArchive() == nil {
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/DataArchiving/SBAActivityArchive.swift]: Nit: Could _our_ version change the method signature to return a BOOL with success? This is confusing.

File: BridgeAppSDK/ORKResult+SBAExtension.swift [hunk 2]
```diff
@@ -280,12 +280,10 @@ class AnswerKeyAndValue: NSObject {
...
 }
 
-protocol ORKQuestionResultAnswerJSON {
+public protocol ORKQuestionResultAnswerJSON {
     func jsonSerializedAnswer() -> AnswerKeyAndValue?
 }
 
-private var _timeOnlyFormatter: NSDateFormatter? = nil
-
 extension ORKQuestionResult: ORKQuestionResultAnswerJSON {
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/ORKResult+SBAExtension.swift:287]: I think this needs to be public, too?

**Comment by syoung-smallwisdom** [comment]
One comment about returning an error rather than a BOOL for success. Otherwise, nice cleanup. Thanks!

File: BridgeAppSDK/ORKResult+SBAExtension.swift (hunk 2 above)
Comment (Erin-Mounts) [inline BridgeAppSDK/ORKResult+SBAExtension.swift:287]: it won't let me declare extensions that implement protocols as public. I assume they match the public-ness of the protocol.

File: BridgeAppSDK/DataArchiving/SBAActivityArchive.swift (hunk 1 above)
Comment (Erin-Mounts) [inline BridgeAppSDK/DataArchiving/SBAActivityArchive.swift]: sure

File: BridgeAppSDK/ORKResult+SBAExtension.swift (hunk 2 above)
Comment (syoung-smallwisdom) [inline BridgeAppSDK/ORKResult+SBAExtension.swift:287]: Ah, right. ORKQuestionResult and  ORKQuestionResultAnswerJSON are public therefore this is automatically public. confusing.

**Comment by syoung-smallwisdom** [comment]
looks good

**Pull request merged**